src/drm/fairplay.js

```diff
@@ -1,10 +1,11 @@
 // @flow
 import BaseDrmProtocol from './base-drm-protocol';
-import Error from '../error/error';
+import type {fairplayDrmConfigType} from '../engines/html5/media-source/adapters/fairplay-drm-handler';
```

Comment:
Start type with upper case and move it to /flow-typed/types folder

Reply:
Fixed